cdflow_commands/deploy.py

```diff
@@ -37,13 +61,30 @@ def _plan(self):
             secrets_file.flush()
             command = self._build_parameters('plan', secrets_file.name)
             logger.debug(f'Running {command}')
-            check_call(
+
+            secrets_to_obfuscate_pattern = '|'.join([
+                re.escape(secret)
+                for secret in self._get_secrets_values()
+            ])
+
+            process = Popen(
                 command, cwd=self._release_path,
                 env=env_with_aws_credetials(
                     os.environ, self._boto_session
                 ),
+                stdout=subprocess.PIPE, stderr=subprocess.PIPE
             )
 
+            while True:
+                (out, err) = process.communicate()
+                self._plan_process_and_print_out(
+                    out, secrets_to_obfuscate_pattern
+                )
+                self._plan_process_and_print_err(err)
```

Comment:
I find these method names a bit confusing i.e. what does `plan process` mean? Looking at the method itself it doesn't seem to match what it does. As this just prints `err`, could it be called `print_err`?